scripts/gaussian_codegen/cdf/emit_test_vectors.py

```
@@ -0,0 +1,264 @@
+
+    indent = "        "
+    rendered = [
+        f"TestCase {{ z_raw: {fmt_u128(z_raw)}, neg: {'true' if neg else 'false'}, expected: {fmt_u128(expected)} }}"
+        for z_raw, neg, expected in cases
+    ]
+    case_lines = f",\n{indent}".join(rendered)
+
+    return f"""{banner}
+/// Deterministic test vectors for `sd29x9_base::cdf`. Each row asserts the
+/// result of `sd29x9::wrap(z_raw, neg).cdf()` matches `expected` to within
+/// `TOLERANCE` raw SD29x9 ULPs (== 5 × 10^-9 absolute).
```

Ruff (0.15.18)

[warning] String contains ambiguous `×` (MULTIPLICATION SIGN). Did you mean `x` (LATIN SMALL LETTER X)?

(RUF001)